owl/EASE-OBJ.owl

```diff
@@ -178,6 +326,27 @@
     
 
 
+    <!-- http://www.ease-crc.org/ont/EASE-OBJ.owl#Cylinder -->
+
+    <owl:Class rdf:about="http://www.ease-crc.org/ont/EASE-OBJ.owl#Cylinder">
+        <rdfs:subClassOf rdf:resource="http://www.ease-crc.org/ont/EASE-OBJ.owl#Shape"/>
+        <rdfs:subClassOf>
+            <owl:Restriction>
+                <owl:onProperty rdf:resource="http://www.ease-crc.org/ont/EASE-OBJ.owl#hasLength"/>
+                <owl:someValuesFrom rdf:resource="http://www.w3.org/2001/XMLSchema#float"/>
+            </owl:Restriction>
+        </rdfs:subClassOf>
+        <rdfs:subClassOf>
+            <owl:Restriction>
+                <owl:onProperty rdf:resource="http://www.ease-crc.org/ont/EASE-OBJ.owl#hasRadius"/>
+                <owl:someValuesFrom rdf:resource="http://www.w3.org/2001/XMLSchema#float"/>
+            </owl:Restriction>
+        </rdfs:subClassOf>
+        <rdfs:comment>A solid geometrical figure with straight parallel sides and a circular or oval cross section.</rdfs:comment>
```

Comment:
If oval, it should have two radii corresponding to the oval's major and minor axis.

Reply:
some values allows having multiple radii, so no problem

Reply:
Then consider "max 2" when dealing with ellipses, and "max 3" when dealing with ellipsoids.

Reply:
I would suggest to have subclasses that specify this